docs/user/openstack/README.md

```diff
@@ -1,51 +1,54 @@
 # OpenStack Platform Support
 
 This document discusses the requirements, current expected behavior, and how to try out what exists so far.
-
 In addition, it covers the installation with the default CNI (OpenShiftSDN), as well as with the Kuryr SDN.
 
-## Kuryr SDN
-
-Kuryr is a CNI plug-in that uses Neutron and Octavia to provide networking for pods and services. It is primarily designed for OpenShift clusters that run on OpenStack virtual machines. Kuryr improves the network performance by plugging
-OpenShift pods into OpenStack SDN. In addition, it provides interconnectivity between OpenShift pods and OpenStack virtual instances.
-
-Kuryr is recommended for OpenShift deployments on encapsulated OpenStack tenant networks in order to avoid double encapsulation, such as running an encapsulated OpenShift SDN over an OpenStack network.
-
-Conversely, using Kuryr does not make sense in the following cases:
-
-* You use provider networks or tenant VLANs.
-* The deployment will use many services on a few hypervisors. Each OpenShift service creates an Octavia Amphora virtual machine in OpenStack that hosts a required load balancer.
-* UDP services are needed.
+## Table of Contents
+
+- [OpenStack Platform Support](#openstack-platform-support)
+  - [Table of Contents](#table-of-contents)
+  - [Reference Documents](#reference-documents)
+  - [OpenStack Requirements](#openstack-requirements)
+    - [Master Nodes](#master-nodes)
+    - [Worker Nodes](#worker-nodes)
+    - [Bootstrap Node](#bootstrap-node)
+    - [Swift](#swift)
+    - [Red Hat Enterprise Linux CoreOS (RHCOS)](#red-hat-enterprise-linux-coreos-rhcos)
+    - [Neutron Public Network](#neutron-public-network)
+  - [OpenStack Credentials](#openstack-credentials)
+  - [Standalone Single-Node Development Environment](#standalone-single-node-development-environment)
+  - [Running The Installer](#running-the-installer)
+    - [Initial Setup](#initial-setup)
+    - [API Access](#api-access)
+      - [Using Floating IPs](#using-floating-ips)
+      - [Without Floating IPs](#without-floating-ips)
+    - [Running a Delpoyment](#running-a-delpoyment)
```

Comment:
Typo right here, @iamemilio

Reply:
this is how we delpoy it!

Reply:
LOL it legit took me 10 minutes to realize what the typo was